### PR TITLE
render glb models on detail page

### PR DESCRIPTION
**Pull request by kaitoo1**

### CODE REVIEW

File: src/scenes/NftDetailPage/NftDetailModel.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+interface ModelViewerJSX {
+  src: string
+  poster?: string
+  class: string
+}
+
+function NftDetailModel({ nft }: Props) {
+  const setContentIsLoaded = useSetContentIsLoaded();
+  setContentIsLoaded()
```
Comment (Robinnnnn) [inline src/scenes/NftDetailPage/NftDetailModel.tsx:31]: `useEffect(setContentIsLoaded, [])`

File: src/scenes/NftDetailPage/NftDetailModel.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+// https://github.com/google/model-viewer/issues/1502
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+    'model-viewer': ModelViewerJSX &
+        React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>
+    }
+  }
+}
```
Comment (Robinnnnn) [inline src/scenes/NftDetailPage/NftDetailModel.tsx:21]: i believe we want to put this in a `global.d.ts` file

Comment (Robinnnnn) [reply src/scenes/NftDetailPage/NftDetailModel.tsx:21]: https://basarat.gitbook.io/typescript/project/modules/globals

Comment (Robinnnnn) [reply src/scenes/NftDetailPage/NftDetailModel.tsx:21]: hmm or maybe not: https://stackoverflow.com/a/42257742/5377437

Comment (kaitoo1) [reply src/scenes/NftDetailPage/NftDetailModel.tsx:22]: resolved on discord

**Pull request merged**